libvcx/src/api/utils.rs

```diff
@@ -130,19 +130,8 @@ pub extern fn vcx_agent_update_info(command_handle: CommandHandle,
     };
 
     spawn(move || {
-        match messages::agent_utils::update_agent_info(&agent_info.id, &agent_info.value) {
-            Ok(()) => {
-                trace!("vcx_agent_update_info_cb(command_handle: {}, rc: {})",
-                       command_handle, error::SUCCESS.message);
-                cb(command_handle, error::SUCCESS.code_num);
-            }
-            Err(e) => {
-                error!("vcx_agent_update_info_cb(command_handle: {}, rc: {})",
-                       command_handle, e);
-                cb(command_handle, e.into());
-            }
-        };
-
+        error!("vcx_agent_update_info is not supported anymore");
```

Comment:
I'd suggest to just remove this function on next iteration (along with removal from FFI wrappers). Besides as this is C-Callable API, this would called through FFI so noone will get to see the deprecation warning.

Reply:
They will see it when building libvcx, though. But I agree in most cases users would download prebuilt binary and use wrapper and FFI